RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -4727,3 +4751,38 @@ float stat_exp_percent(stat s)
 	}
 	return 0;
 }
+
+int auto_roughExpectedTurnsLeftToday()
```

Comment:
needs to cover more paths:
* boozetafarian (no eat_val)
* teet (no drink_val)
* oxy (no eat or drink)
* jarlsberg (set drink_val to 1; jarlsberg only drinks his own booze / astral pilsners, and if you don't have pilsners they're not very good)
* KOLHS (set drink_val to 2.5?)

other paths that have custom stuff but are okay
* zombie slayer (good brain is 3)
* nuclear autumn (you can only consume small stuff but have low organs)
* license to adventure (I think you can get martinis to 3.5 adv/drunk)
* gelatinous noob (no organs, but the function is still an underestimate)
* you, robot (no organs but again an underestimate is fine)
* grey you (same)

Reply:
Thanks for the list, I'll check them. But note that paths that can't eat/drink are handled naturally by the check on organs left. Eg for a teetotaller, inebriety_left() will always be zero so we don't have to worry about the dirnk_val.